stl/inc/__msvc_minmax.hpp

```diff
@@ -0,0 +1,86 @@
+// __msvc_minmax.hpp internal header (core)
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef __MSVC_MINMAX_HPP
+#define __MSVC_MINMAX_HPP
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+#include <stdint.h>

```

Comment:
I guess it would be preferred to include `<cstdint>` instead of `<stdint.h>`, see #1405.
(Pedantically speaking, `(u)intN_t` should be `_STD`-qualified if we include `<cstdint>`. But in MSVC STL it can be guaranteed that `::(u)intN_t` is available.)
```suggestion
#include <cstdint>
```

Reply:
I can actually spell out them via keywords, as done in atomics and avoid include

Reply:
Ok, I'll commit as suggested, because in the `.cpp` I use these standard integer types too